src/data/equiv/functor.lean

```diff
@@ -7,15 +7,31 @@ Authors: Johan Commelin, Simon Hudon, Scott Morrison
 import data.equiv.basic
 import category.bifunctor
 
+/-!
+# Functor and bifunctors can be applied to `equiv`s.
+
+We define
+```
+def functor.map_equiv (f : Type u → Type v) [functor f] [is_lawful_functor f] :
+  α ≃ β → f α ≃ f β
+```
+and
+```
```

Comment:
````suggestion
```lean
def functor.map_equiv (f : Type u → Type v) [functor f] [is_lawful_functor f] :
  α ≃ β → f α ≃ f β
```
and
```lean
````
because someday (soon after pygments has its next official release) we'll have Lean syntax highlighting in the docs.

Reply:
Thanks!

Reply:
Actually, can we not tell pygments to globally use `lean` as the default language? This just seems like noise if it has to be everywhere the documentation.

Reply:
Yeah, we could. There are definitely some snippets (e.g. example output) where we won't want Lean highlighting, but maybe those are few enough that it doesn't matter.